[2024-08-08T15:03:19.619+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-08T15:03:19.638+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:02:59.974409+00:00 [queued]>
[2024-08-08T15:03:19.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:02:59.974409+00:00 [queued]>
[2024-08-08T15:03:19.643+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-08T15:03:19.654+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): limpieza> on 2024-08-08 15:02:59.974409+00:00
[2024-08-08T15:03:19.661+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=95) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-08T15:03:19.661+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_prueba_tecnica', 'limpieza', 'manual__2024-08-08T15:02:59.974409+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/dag_prueba_tecnica.py', '--cfg-path', '/tmp/tmp93by5zs4']
[2024-08-08T15:03:19.662+0000] {standard_task_runner.py:64} INFO - Started process 108 to run task
[2024-08-08T15:03:19.662+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask limpieza
[2024-08-08T15:03:19.694+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:02:59.974409+00:00 [running]> on host aa12b50c95f1
[2024-08-08T15:03:19.757+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='miguel_guimarey' AIRFLOW_CTX_DAG_ID='dag_prueba_tecnica' AIRFLOW_CTX_TASK_ID='limpieza' AIRFLOW_CTX_EXECUTION_DATE='2024-08-08T15:02:59.974409+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-08T15:02:59.974409+00:00'
[2024-08-08T15:03:19.758+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-08T15:03:22.814+0000] {warnings.py:112} WARNING - /opt/***/dags/dag_prueba_tecnica.py:40: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(data, orient='records')

[2024-08-08T15:03:32.637+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-08-08T15:04:09.152+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-08T15:04:09.153+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-08T15:04:09.162+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_prueba_tecnica, task_id=limpieza, run_id=manual__2024-08-08T15:02:59.974409+00:00, execution_date=20240808T150259, start_date=20240808T150319, end_date=20240808T150409
[2024-08-08T15:04:09.311+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-08T15:04:09.324+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-08T15:04:09.327+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
